content/en/armory-enterprise/armory-admin/clouddriver-sql-configure.md

```diff
@@ -26,6 +26,12 @@ You can skip this step if you create the database during provisioning - for inst
 
 Once you've provisioned your RDBMS and ensured connectivity with Spinnaker, you need to create the database:
 
+{{% alert title="Important" color=warning %}} 
+ Configure the RDBMS driver exactly as described in [Set up Clouddriver to use SQL - Database Setup](https://spinnaker.io/docs/setup/productionize/persistence/clouddriver-sql/#database-setup). The MySQL database schema must be configured to with:
+ - Default character set `utfmb4`
+ - Default collate `utf8mb4_unicode_ci`
+  {{% /alert %}}
+
```

Comment:
@tsonny would it be a good idea to separate this out into its own markdown file to include elsewhere? Asking because I see the same text in the dinghy-enable.md file, want to ensure that in the future, we don't make changes in one spot that aren't reflected in the other

Reply:
I was going to suggest the same thing for the same reason. @tsonny I can show you how to create an included component, or you can take a look at https://armory.slab.com/posts/advanced-info-p1rlytq7#h794b-reusing-text